src/tye/Program.RunCommand.cs

```diff
@@ -51,7 +51,8 @@ private static Command CreateRunCommand(string[] args)
 
             command.AddOption(new Option("--debug")
             {
-                Description = "Wait for debugger attach in all services.",
+                Argument = new Argument<string>("services|\"all\""),
```

Comment:
I'd suggest making this multiple arguments instead of comma-delimited. 

```
--debug foo --debug bar
```

We have the technology. We also might want to use `*` as a sentinel instead of `all` since I GUESS `all` is a valid service name.